index.d.ts

```diff
@@ -346,13 +346,27 @@ type OnwardsType = {
     trails: TrailType[];
 };
 
+interface CommercialConfigType {
+    isPaidContent?: boolean;
+    pageId: string;
+    webPublicationDate?: number;
```

Comment:
Not sure if this matters or not but I've only ever seen `webPublicationDate` as a string. I've heard rumours that we have dates in two different formats in CAPI so maybe this is what we're seeing here? If so, is it possible to force it to a string on the server? Or maybe use a different name?

Reply:
I hear you and I've noticed that before too ... the webPublicationDate comes in the page config as a number .. Don't think it's up to me to change stuff as who knows what that could break 🤷‍♂️